### ISSUE TITLE
Unknown function called: lookup

### ISSUE THREAD

**Issue by R3n3gad3WT** [wata727/tflint#187]
Hi, 

After running tflint I get this error: 
- Evaluation error: 1:31: unknown function called: lookup in main_cdn_group5.tf:112

I'm running the following line: 
```HCL
dns_domain_name      = "${var.environment == "prod" ? lookup(var.dns_domain_name, var.group5_real_url) : lookup(var.dns_domain_name, var.environment)}"
```
I use lookup in this file multiple times, and this is the only time Tflint isn't able to recognize lookup  as a built-in function. Is this due to being used in a Conditional?

**Comment by alivespirit** [comment]
Got similar error here, multiple lookups like this work:
`${lookup(var.internal_dns_zone_id, "${var.account}-${var.region}")}`
but one lookup fails `${lookup(var.default_ami_map, "${var.account}-${var.region}")}`.
Could get it work only when changed latter to `${var.default_ami_map["${var.account}-${var.region}"]}`

**Comment by wata727** [comment]
Hi there,
TFLint cannot evaluate the `lookup` function, but the feature of ignoring it is broken.
Conditional is handled correctly in TFLint as well.
